identifiers/country-fr/oversea.csv

```diff
@@ -0,0 +1,12 @@
+id,name
+ocd-division/country:fr/oversea_collectivity:mf,St-Barthélemy/St-Martin
+ocd-division/country:fr/oversea_collectivity:nc,Nouvelle-Calédonie
```

Comment:
Since 1999, New Caledonia is not an oversea collectivity. This OCDID is only correct if we intend to refer to that division prior to 1999. From https://en.wikipedia.org/wiki/Overseas_France:

> New Caledonia is neither an overseas region nor an overseas collectivity

So, use `sui_generis_collectivity`.
 

Reply:
Done.